.github/workflows/build-test.yml

```diff
@@ -48,11 +48,15 @@ jobs:
   
     - name: Configure
       shell: bash -l {0}
+      env:
+        CXXFLAGS: ""
       run: |
         python ./setup --type=$BUILD_TYPE --generator=Ninja --prefix=$GITHUB_WORKSPACE/Software/VAMPyR build --cmake-options="-DCMAKE_VERBOSE_MAKEFILE=ON"
 
     - name: Build
       shell: bash -l {0}
+      env:
+        CXXFLAGS: ""
```

Comment:
Do we build with C++17 now?

Reply:
I think VAMPyR is inheriting the build flags from MRCPP.

Reply:
yes, but the conda env is still a mess

Reply:
I think it looks fine in a local build without conda